src/components/Header/ChainConnectivityWarning.tsx

```diff
@@ -0,0 +1,72 @@
+import { Trans } from '@lingui/macro'
+import { CHAIN_INFO, L2ChainInfo } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+import { ExternalLink, MEDIA_WIDTHS } from 'theme'
+
+const BodyRow = styled.div`
+  color: black;
+  font-size: 12px;
+`
+const CautionIcon = styled(AlertOctagon)`
+  color: black;
+`
+const Link = styled(ExternalLink)`
+  color: black;
```

Comment:
prefer `theme.black` so this color is tracked with our theme. It's still defined as `black` for both light and dark but if we ever change it later we'll know